server/models/feed.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Contains the User struct, as well as any methods*/
+package models
+
+
+//All strings for now, type will be changed as project develops
+type Feed struct {
+	OwnerID  string
+	ImageURL []string
```

Comment:
Would be easier if you also added the owner's name and profile image url to the output as well, instead of running a get for every user on the frontend

Reply:
Yes. Will do. I wanted to make sure the body is good. :-)